packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -172,7 +173,7 @@ export class PharosImageCard extends LitElement {
           label="More actions"
           @click=${this._handleClick}
         ></pharos-button>`
-      : nothing;
+      : html`<div class="card__button"><slot name="action-menu"></slot></div>`;
```

Comment:
Wondering if we should name it `action-button` since that is the expected content to be passed (and the menu can exist outside the card)?

Reply:
Hmm I'm open for that. I'll give this until tomorrow morning for other opinions before I make the change?

Reply:
I might suggest `class="card__action-button"` and align that with `slot name="action-button"`

Reply:
I'm leaning towards `action-button`, but feel like having a consistent `action-menu` might be more dev friendly.

Reply:
👀 updating to `action` button and can make the BEM class modification suggestion too. 👍

Reply:
Done! https://github.com/ithaka/pharos/pull/128/commits/960d357bb3a9232ca5019d69d1f054be8b292e0d